src/views/pacbio/PacbioRunIndex.vue

```diff
@@ -113,12 +113,6 @@ export default {
         { key: 'id', label: 'Run ID', sortable: true, tdClass: 'run-id' },
         { key: 'name', label: 'Name', sortable: true, tdClass: 'name' },
         { key: 'state', label: 'State', sortable: true, tdClass: 'state' },
```

Comment:
Check whether SKBB to be shown in run table

Reply:
@BenTopping

Reply:
@BenTopping how far did you get with this?

Reply:
From Ben:

```
const fetchPacbioRuns = async ({ commit, getters }, filter) => {
  const request = getters.runRequest
  const promise = request.get({
    filter,
    include: 'plates', 
  })
  const response = await handleResponse(promise)

  const { success, data: { data, included = [] } = {}, errors = [] } = response
  

  if (success) {
    let { plates } = groupIncludedByResource(included)
    plates = dataToObjectById(plates)
    data.map(run => {
      const sequencing_kit_box_barcodes = run.relationships.plates.data.map((plate) => {
        if (plates[plate.id]) {
          return `Plate ${plates[plate.id].plate_number}: ${plates[plate.id].sequencing_kit_box_barcode}`
        }
      }).join(', ')
      run.attributes.sequencing_kit_box_barcodes = sequencing_kit_box_barcodes
    })
    commit('setRuns', data)
  }

  return { success, errors }
}
```
![Uploading Screenshot 2023-06-23 at 10.54.36.png…]()

Reply:
@harrietc52